ui/Screen/IdentityCreation.svelte

```diff
@@ -0,0 +1,175 @@
+<script>
+  import { gql } from "apollo-boost";
+  import { getClient, mutate } from "svelte-apollo";
+  import { pop, push } from "svelte-spa-router";
+  import validatejs from "validate.js";
+
+  import {
+    IMAGE_FILENAME,
+    NAME_MATCH,
+    SINGLE_WORD_MATCH
+  } from "../lib/validationHelpers.js";
+  import { setCurrentIdentity } from "../store/identity.js";
+  import { showNotification } from "../store/notification.js";
+
+  import { Button, Input, Text, Title } from "../DesignSystem/Primitive";
+  import { ModalLayout } from "../DesignSystem/Component";
+
+  let handle,
+    displayName,
+    avatarUrl,
+    validations,
+    beginValidation = false;
+
+  validatejs.options = {
+    fullMessages: false
+  };
+
+  const constraints = {
+    handle: {
+      presence: {
+        message: "You must provide a handle",
+        allowEmpty: false
+      },
+      format: {
+        pattern: SINGLE_WORD_MATCH,
+        message: "Handle should match [a-z0-9][a-z0-9_-]+"
+      }
+    },
+    displayName: {
+      format: {
+        pattern: NAME_MATCH,
+        message: "Display name should match [a-z0-9]"
+      }
+    },
+    avatarUrl: {
```

Comment:
Could also use the built-in matcher to keep it simple:
```
    imageUrl: {
      optional: {
        url: {
          schemes: ["http", "https"],
          message: "Not a valid avatar URL",
          allowLocal: false
        }
      }
    },
```

We already do it like this in `CreateProject.svelte`.